simulators/portal-interop/src/main.rs

```diff
@@ -561,6 +560,9 @@ dyn_async! {
             Err(err) => panic!("{}", &err.to_string()),
         }
 
+        // send a ping so client A sees it as a seen/connected node
+        let _ = client_a.rpc.ping(target_enr.clone()).await;
```

Comment:
Should we not verify the result to see if the message arrived and was responded?